usage/bigquery.go

```diff
@@ -160,6 +179,48 @@ func buildUsageSummaryQuery(table string, userID string, creatorID string, spec
 	return sql, args, nil
 }
 
+func buildUsageUsersQuery(table string, userIDs []string, spec QuerySpec) (string, []interface{}, error) {
+	if len(userIDs) == 0 {
+		return "", nil, fmt.Errorf("userIDs cannot be empty")
+	}
+
+	placeholders := make([]string, len(userIDs))
+	args := make([]interface{}, len(userIDs))
+
+	for i, userID := range userIDs {
+		placeholders[i] = "?"
+		args[i] = userID
+	}
+
+	query := squirrel.Select(
+		"user_id",
+		"cast(sum(transcode_total_usage_mins) as FLOAT64) as transcode_total_usage_mins",
+		"cast(sum(delivery_usage_mins) as FLOAT64) as delivery_usage_mins",
+		"cast((sum(storage_usage_mins) / count(distinct usage_hour_ts)) as FLOAT64) as storage_usage_mins").
+		From(table).
+		Where(fmt.Sprintf("user_id IN (%s)", strings.Join(placeholders, ",")), args...).
```

Comment:
Neat! Tho I still think just querying for all users is simpler and would avoid table scans in our main DB (and still won't do table scans on bigquery since we filter by timestamp)